index.html

```diff
@@ -431,7 +440,7 @@
 
       <p>Pointer lock must be exited if the
       <a>target</a> is removed from its document, or the <a>user agent</a>,
-      window, or tab loses focus. Moving focus between elements of
+      window, or tab loses focus. Moving focus between elements whose shadow-including root is
```

Comment:
Hmm, my change seems wrong.

The sentence looks meaning that moving focus from one active document to another, and they can be in different browsing context (e.g. main page and iframe).  So the plural form of "active document" should be fine.

So I don't think change is necessary here.